src/api/task/api/dto/task.go

```diff
@@ -1,51 +1,62 @@
 package dto
 
 import (
+	"proyectos/src/api/constants"
+	"proyectos/src/api/errors"
 	"proyectos/src/api/task/domain/model"
 	"time"
 )
 
 // Task dto
 type Task struct {
 	ID             int64     `validate:"gt=0" json:"id"`
-	Name           string    `validate:"required,min=2,max=50" json:"name"`
+	Name           string    `validate:"required,min=2,max=500" json:"name"`
 	Description    string    `validate:"required" json:"description"`
 	StartDate      time.Time `validate:"required" json:"start_date"`
-	HoursWorked    int       `validate:"required" json:"hours_worked"`
+	WorkedHours    int       `validate:"required" json:"worked_hours"`
 	EstimatedHours int       `validate:"required" json:"estimated_hours"`
-	ProjectId      int64     `validate:"required" json:"project_id"`
+	ProjectID      int64     `validate:"required" json:"project_id"`
 	State          string    `validate:"required" json:"state"`
 	CreationDate   time.Time `validate:"required" json:"creation_date"`
 	AssignedTo     string    `validate:"required" json:"assigned_to"`
```

Comment:
Is it not a possibility to create a task without assigning it to someone? If it's possible we shouldn't make it a required field and we should default to null

Reply:
makes sense